ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/route_maps.py

```diff
@@ -29,7 +31,7 @@ def route_maps(self) -> dict | None:
 
         if self._overlay_routing_protocol == "ebgp":
             if self._evpn_prevent_readvertise_to_server is True:
-                remote_asns = sorted(list(set(self._evpn_route_servers[route_server].get("bgp_as") for route_server in self._evpn_route_servers)))
+                remote_asns = natural_sort(list(set(self._evpn_route_servers[route_server].get("bgp_as") for route_server in self._evpn_route_servers)))
```

Comment:
I think natural_sort will work fine with the `set` directly, so you can skip the list if you want. Not important.